addons/addon-base-raas/packages/.pnpm-debug.log

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
don't add those debugging log files to SCM, you might add it to `.gitignore` to exclude it

Reply:
Done.